src/scripts/main.js

```diff
@@ -2,7 +2,36 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
```

Comment:
If you want to change data of each person, you should use `map` method instead and store it's returned array with changes